system-variables.md

```diff
@@ -271,6 +271,14 @@ Constraint checking is always performed in place for pessimistic transactions (d
 - Use a bigger value in OLAP scenarios, and a smaller value in OLTP scenarios.
 - For OLAP scenarios, the maximum value cannot exceed the number of CPU cores of all the TiKV nodes.
 
+### tidb_dml_batch_size
+
+- Scope: SESSION
+- Default value: 0
+- Example value: 20000
```

Comment:
@nullnotnil Could you explain what `example value` is? I find it confusing when reviewing the translation of this PR. From the context, `20000` does not appear in this document but in `sql-statement-load-data.md `. Did you mean the `example` in `sql-statement-load-data.md `?

Reply:
I picked `20000` as an example value because this is the prior batch size for `LOAD DATA` before it was changed to not be batched in master.

I think it is potentially useful for other sysvars, because the min/max values may be a very wide range.

Reply:
Can we say `In earlier releases of TiDB, the default batch size for LOAD DATA was 20000`? Here the `example` seems a little confusing.

Reply:
Let's just remove it. I don't think people will know to look up this value when they get an error, they will look up the reference for `LOAD DATA`, which mentions the previous default.

Reply:
Removed in https://github.com/pingcap/docs/pull/3977.